config/initializers/decidim_menu.rb

```diff
@@ -0,0 +1,51 @@
+Decidim.menu :menu do |menu|
```

Comment:
One more thing I noticed with this is that in some mid-sizes all the links are not visible:
![Nav hidden](https://user-images.githubusercontent.com/864340/194512116-e1853838-ddda-4787-aed6-955246aeb566.png)

With this CSS change:
![CSS change](https://user-images.githubusercontent.com/864340/194512197-3ad59edd-64cd-46cc-8dbc-c4d446e8719e.png)

It would work better:
![Less padding in menu](https://user-images.githubusercontent.com/864340/194512275-716b55bb-937a-4075-a735-cc88623250d1.png)

---

That said, I don't know how big of a problem that is really because we have the same issue even with the current navigation in the live versions of Metadecidim:
![Current navigation](https://user-images.githubusercontent.com/864340/194512500-d68c4b30-d7e0-47f3-a6d5-6348e0ff6474.png)

Reply:
Fixed in beb306a26e6df9b67173ce05a6f23fe0d20b7eb0